index.js

```diff
@@ -68,17 +68,13 @@ function getUnixLocaleSync() {
 }
 
 function getWinLocale() {
-	return execa.stdout('wmic', ['os', 'get', 'locale'])
-		.then(stdout => {
-			const lcidCode = parseInt(stdout.replace('Locale', ''), 16);
-			return lcid.from(lcidCode);
-		});
+	return execa.stdout(path.join(__dirname, 'locale.exe'))
+		.then(stdout => stdout.replace(/-/, '_'));
```

Comment:
The `replace` here is no longer needed: https://github.com/sindresorhus/os-locale/commit/2a3fccb952db4ba64df025dc04a5467a6d42cf42